samples/menus/nav-drawer/styling/src/drawer-styling.css

```diff
@@ -0,0 +1,16 @@
+igc-nav-drawer::part(base) {
+    background: #2d313a;
+}
+
+igc-nav-drawer-item::part(base) {
+    color: #fff;
+}
+
+igc-nav-drawer-item[active]::part(base) {
```

Comment:
I would suggest to change the hover color of the drawer item since the default one is too bright.